api/nakadi-event-bus-api.yaml

```diff
@@ -317,7 +345,7 @@ paths:
           required: true
         - name: batch_limit
           in: query
-          description: Maximum number of `SimpleStreamElement`s in each chunk of the stream. If unspecified assumes default value 1 (i.e. each event is individually submitted).
+          description: Maximum number of `EventStreamAtom`s in each chunk of the stream. If unspecified assumes default value 1 (i.e. each event is individually submitted).
```

Comment:
@valgog Yes, it's a number of events in a batch. And there's really a problem with description, because it's not a number of `EventStreamAtom`s. It's a number of events in `EventStreamAtom`.